components/layout/display_list/items.rs

```diff
@@ -986,19 +995,16 @@ impl WebRenderImageInfo {
 /// The type of the scroll offset list. This is only populated if WebRender is in use.
 pub type ScrollOffsetMap = HashMap<ExternalScrollId, Vector2D<f32>>;
 
-
 pub trait SimpleMatrixDetection {
     fn is_identity_or_simple_translation(&self) -> bool;
 }
 
 impl SimpleMatrixDetection for LayoutTransform {
     #[inline]
     fn is_identity_or_simple_translation(&self) -> bool {
-        let (_0, _1) = (Zero::zero(), One::one());
-        self.m11 == _1 && self.m12 == _0 && self.m13 == _0 && self.m14 == _0 &&
-        self.m21 == _0 && self.m22 == _1 && self.m23 == _0 && self.m24 == _0 &&
-        self.m31 == _0 && self.m32 == _0 && self.m33 == _1 && self.m34 == _0 &&
-        self.m44 == _1
+        let (_0, _1) = (0.0, 1.0);
```

Comment:
This looks unrelated, and all the unrelated changes makes this somewhat hard to review.

Reply:
Sorry about making review really hard by rustfmting everything.